mlflow/gateway/fluent.py

```diff
@@ -132,6 +132,57 @@ def delete_route(name: str) -> None:
     MlflowGatewayClient().delete_route(name)
 
 
+@experimental
+def set_limits(route: str, limits: LimitsConfig) -> LimitsConfig:
```

Comment:
I changed to forcing users to pass  limits as `List[Dict[str, any]]`, I think it makes more sense to pass dictionary as payload instead of letting users constructing LimitsConfig.

Reply:
great catch, thanks!